components/encryption/src/encrypted_file/mod.rs

```diff
@@ -45,6 +48,10 @@ impl<'a> EncryptedFile<'a> {
                 encrypted_content.merge_from_bytes(content)?;
                 let plaintext = master_key.decrypt(&encrypted_content)?;
 
+                ENCRYPT_DECRPTION_FILE_HISTOGRAM
```

Comment:
can we distinguish metrics for `key_dict` read/write and `file_dict` read/write?

Reply:
Fixed